pywhispercpp/model.py

```diff
@@ -261,7 +261,7 @@ def __call_new_segment_callback(ctx, n_new, user_data) -> None:
         n = pw.whisper_full_n_segments(ctx)
         start = n - n_new
         res = Model._get_segments(ctx, start, n)
-        Model._new_segment_callback(res)
+        Model._new_segment_callback(res[0])
```

Comment:
Actually, `_get_segments` returns a `List[Segment]`, which may contain more than one segment depending on the value of `n_new`. 
So, the correct approach is to iterate over the `res` list and call `Model._new_segment_callback` for each of the resulting segments. 
Please update it to use a loop, test the changes, and inform me of the results.

Reply:
Changed it to `[Model._new_segment_callback(segment) for segment in res]` (if you don't like it written that way, I can just turn it into a regular for loop, but I think this is easily readable), and it continues to work fine, but should now handle multiple segments in one response.

Reply:
Yes, it's readable, but there is no need to create an extra List object, we just need to call the function. A simple for loop is better in my opinion.

Reply:
That makes sense, updated.

Reply:
Perfect, thanks for the contribution :)